.pre-commit-config.yaml

```diff
@@ -11,10 +11,19 @@ repos:
     hooks:
       - id: helm-docs
   - repo: https://github.com/bridgecrewio/checkov.git
-    rev: 2.3.3
+    rev: 2.3.130
     hooks:
       - id: checkov
         files: \.yaml$
         args:
         - --quiet
         - --compact
+  - repo: https://github.com/jtyr/kubeconform-helm
+    rev: v0.1.12
+    hooks:
+      - id: kubeconform-helm
+        files: ^charts/[^/]+/(\.kubeconform|\.helmignore|templates/NOTES.txt|.*\.(ya?ml|json|tpl))$
+        args:
+          - --kubernetes-version=1.25.0
+          - --exclude-charts=alfresco-common,alfresco-sync-service
+          - --values-dir=ci
```

Comment:
I tried using that hook previously but decided to stay away from it as it failed miserably (at least on my mac and on my linux machine) with the error bellow:
```
Kubeconform Helm.........................................................Failed
- hook id: kubeconform-helm
- exit code: 1

Traceback (most recent call last):
  File "/Users/Alexandre.Chapellon@hyland.com/.cache/pre-commit/repogk2ka3dv/scripts/pre-commit.py", line 9, in <module>
    import plugin_wrapper as pw
  File "/Users/Alexandre.Chapellon@hyland.com/.cache/pre-commit/repogk2ka3dv/scripts/plugin_wrapper.py", line 8, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

which made this hook look broken to me

Reply:
actually fixed